playground/nextjs/pages/session-examples/index.tsx

```diff
@@ -49,7 +49,7 @@ const SessionExamplesPage: NextPage = () => {
       <PublicMetadataWithHook />
       <PublicMetadataClassHOC />
       <PublicMetadataFnHOC />
-      <PublicMetadataFaaC />
+      {/*<PublicMetadataFaaC />*/}
```

Comment:
Why do we comment this out?

Reply:
I think the HoC has been removed, @panteliselef said he is going to take a look at it later and maybe totally remove it.